modules/lvms-installing-logical-volume-manager-operator-disconnected-environment.adoc

```diff
@@ -0,0 +1,25 @@
+// Module included in the following assemblies:
+//
+// storage/persistent_storage/persistent_storage_local/persistent-storage-using-lvms.adoc
+
+:_content-type: PROCEDURE
+[id="lvms-installing-lvms-disconnected-env_{context}"]
+= Installing {lvms} in a disconnected environment
+
+You can install {lvms} on {product-title} {product-version} in a disconnected environment by following the procedures in the _About disconnected installation mirroring_ section.
+
+.Prerequisites
+
+* You read the _About disconnected installation mirroring_ section.
+* You have access to the {product-title} image repository.
+* You created a mirror registry.
+
+.Procedure
+
+. Follow the steps in the _Creating the image set configuration_ procedure. To create an `ImageSetConfiguration` for {lvms}, you can use the following example YAML file:
```

Comment:
Would it make it easier to keep the reader here and not reference the other section? Maybe another preq: 

* Create a template for the image set configuration, as described in the _Creating the image set configuration_ procedure.

Reply:
I would keep it as a step is because we have an example config file specific for LVM Storage and it seems more practical to keep it there. WDYT?